src/sidebar/components/test/annotation-omega-test.js

```diff
@@ -221,6 +224,67 @@ describe('AnnotationOmega', () => {
     });
   });
 
+  describe('reply thread toggle button', () => {
+    const findRepliesButton = wrapper =>
+      wrapper.find('Button').filter('.annotation-omega__reply-toggle');
+
+    it('should render a toggle button if the annotation has replies', () => {
+      fakeMetadata.isReply.returns(false);
+      const wrapper = createComponent({
+        replyCount: 5,
+        threadIsCollapsed: true,
+      });
+
+      assert.isTrue(findRepliesButton(wrapper).exists());
+      assert.equal(
+        wrapper.find('Button').props().buttonText,
```

Comment:
Can I suggest it would be slightly clearer to only look for the button once. Something like:

```js
const replyButton = findRepliesButton(wrapper);
assert.isTrue(replyButton.exists());
assert.equal(replyButton.props().buttonText, 'Show replies (5)')
```